Specs/Number/Korean/NumberModel.json

```diff
@@ -1,6 +1,20 @@
 [{
+  "Input": "마이너스일만",
+  "NotSupported": "javascript,python,java",
+  "Results": [
+    {
+      "Text": "마이너스일만",
+      "TypeName": "number",
+      "Resolution": {
+        "value": "-10000"
+      }
+    }
+  ]
+}
+,
+  {
   "Input": "192.",
-  "NotSupportedByDesign": "dotNet,javascript,python,java,",
+  "NotSupported": "javascript,python,java",
```

Comment:
If you keep the ...ByDesign, it will avoid warnings being thrown when these don't run.